nameko/standalone/events.py

```diff
@@ -31,6 +34,11 @@ def event_dispatcher(nameko_config, **kwargs):
     def dispatch(service_name, event_type, event_data):
         conn = Connection(nameko_config[AMQP_URI_CONFIG_KEY])
 
+        if KOMBU_SERIALIZER_CONFIG_KEY in nameko_config.keys():
+            serializer = nameko_config[KOMBU_SERIALIZER_CONFIG_KEY]
+        else:
+            serializer = DEFAULT_KOMBU_SERIALIZER
```

Comment:
`foo.get(key) or default` is different to `foo.get(key, default)` if e.g. `foo[key] == None  # or 0`